mono/metadata/metadata.c

```diff
@@ -7824,3 +7826,262 @@ mono_type_set_amods (MonoType *t, MonoAggregateModContainer *amods)
 	g_assert (t_full->mods.amods == NULL);
 	t_full->mods.amods = amods;
 }
+
+static void
```

Comment:
Probably everything here should be under `#ifndef DISABLE_COM` for the configurations where we don't compile in the COM support.  (Pretty much all the AOT platforms, for example)

Reply:
I'm going to make RuntimeType.GUID use it too, so I suppose that's not going to be only for COM anymore.

Reply:
That's not what we want, for COM disabled configurations it needs to return empty GUID